src/screens/SendScanQRCode.js

```diff
@@ -57,10 +57,7 @@ class SendScanQRCode extends React.Component {
     if (!qrcode.isValid) {
       this.showAlertError(qrcode.error);
     } else if (qrcode.token && qrcode.amount) {
-      const isTokenRegistered = this.props.tokens.some(
-        (stateToken) => stateToken.uid === qrcode.token.uid
-      );
-      if (isTokenRegistered) {
+      if (qrcode.token.uid in this.props.tokens) {
```

Comment:
suggestion(non-blocking): Add a comment explaining that this is a search into the registered tokens.

Otherwise the reader would have to search what the `props.tokens` mean to understand what this rule is about. This was made clearer before by the old variable name itself.